src/SortMovies/SortMovies.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, useState } from 'react';
```

Comment:
Stay away from hooks for this project, I promise you can use them in the next one.